.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*.{adoc, bat, groovy, html, java, js, jsp, kt, kts, md, properties, py, rb, sh, sql, svg, txt, xml, xsd}]
+charset = utf-8
+
+[*.{groovy, java, kt, kts, xml, xsd}]
+indent_style = space
+indent_size = 4
+continuation_indent_size = 8
```

Comment:
Can we try `continuation_indent_size = 4`, maybe it will end up with less lines changed